backend/src/database/migrations/20230608034909-create-govsg-tables.js

```diff
@@ -229,7 +229,7 @@ module.exports = {
       },
       recipient: {
         type: Sequelize.DataTypes.STRING(255),
-        allowNull: true,
+        allowNull: false,
```

Comment:
should be false because every tx message will have a recipient.

have edited the migration file directly.